beginner_source/fgsm_tutorial.py

```diff
@@ -447,4 +447,8 @@ def test( model, device, test_loader, epsilon ):
 # implement a different attack from the NIPS 2017 competition, and see how
 # it differs from FGSM. Then, try to defend the model from your own
 # attacks.
-# 
+#
+# A further direction to go, depending on available resources, is to modify
+# the code to support processing work in batch, in parallel, and or distributed
+# vs. working on one attack at a time in the above for each epsilon test() loop.
```

Comment:
Do you want to add examples of those?

Reply:
thought about it and was thinking this is more a beginner example and appropriate for this section.. Perhaps a second example addressing one or more of the above concerns is warranted and we could link from here?